quickwit-config/src/source_config.rs

```diff
@@ -162,10 +162,11 @@ pub struct KafkaSourceParams {
     pub client_params: serde_json::Value,
 }
 
-#[doc(hidden)]
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct KinesisSourceParams {
-    stream_name: String,
+    pub stream_name: String,
+    pub region: Option<String>,
```

Comment:
I can open an issue for using an enum rather than two options here.

Reply:
Yes enum would be better.